.github/workflows/ci.yaml

```diff
@@ -49,7 +49,7 @@ jobs:
         wget https://packages.osrfoundation.org/gazebo.gpg -O /usr/share/keyrings/pkgs-osrf-archive-keyring.gpg
         echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/pkgs-osrf-archive-keyring.gpg] http://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -cs) main" | tee /etc/apt/sources.list.d/gazebo-stable.list > /dev/null
         if [ "$GZ_VERSION" == "garden" ]; then
-          export GZ_DEPS="libgz-sim7-dev libgz-plugin2-dev"
+          export GZ_DEPS="libgz-sim7-dev libgz-plugin2-dev gz-sim7-cli"
```

Comment:
I think we should start using rosdep for these steps instead to save us some headache on the buildfarm later